2019-08-05 17:01:15.404  INFO 19548 --- [main] cn.kang.Application                      : Starting Application on PS20190426HEJH with PID 19548 (D:\workSpace\IDEACode\my_netty\target\classes started by Admin in D:\workSpace\IDEACode\my_netty)
2019-08-05 17:01:15.407  INFO 19548 --- [main] cn.kang.Application                      : No active profile set, falling back to default profiles: default
2019-08-05 17:01:16.251  INFO 19548 --- [main] o.s.b.w.embedded.tomcat.TomcatWebServer  : Tomcat initialized with port(s): 8888 (http)
2019-08-05 17:01:16.265  INFO 19548 --- [main] o.apache.catalina.core.StandardService   : Starting service [Tomcat]
2019-08-05 17:01:16.265  INFO 19548 --- [main] org.apache.catalina.core.StandardEngine  : Starting Servlet engine: [Apache Tomcat/9.0.19]
2019-08-05 17:01:16.327  INFO 19548 --- [main] o.a.c.c.C.[Tomcat].[localhost].[/]       : Initializing Spring embedded WebApplicationContext
2019-08-05 17:01:16.327  INFO 19548 --- [main] o.s.web.context.ContextLoader            : Root WebApplicationContext: initialization completed in 870 ms
2019-08-05 17:01:16.370  INFO 19548 --- [main] cn.kang.config.RedisConfig               : JedisPool注入成功！！
2019-08-05 17:01:16.371  INFO 19548 --- [main] cn.kang.config.RedisConfig               : redis地址：114.116.175.209:6379
2019-08-05 17:01:16.744  INFO 19548 --- [main] .s.a.AnnotationAsyncExecutionInterceptor : No task executor bean found for async processing: no bean of type TaskExecutor and no bean named 'taskExecutor' either
2019-08-05 17:01:16.760  INFO 19548 --- [main] o.s.b.w.embedded.tomcat.TomcatWebServer  : Tomcat started on port(s): 8888 (http) with context path ''
2019-08-05 17:01:16.762  INFO 19548 --- [main] cn.kang.Application                      : Started Application in 1.721 seconds (JVM running for 2.371)
2019-08-05 17:04:20.385  INFO 19548 --- [http-nio-8888-exec-1] o.a.c.c.C.[Tomcat].[localhost].[/]       : Initializing Spring DispatcherServlet 'dispatcherServlet'
2019-08-05 17:04:20.385  INFO 19548 --- [http-nio-8888-exec-1] o.s.web.servlet.DispatcherServlet        : Initializing Servlet 'dispatcherServlet'
2019-08-05 17:04:20.390  INFO 19548 --- [http-nio-8888-exec-1] o.s.web.servlet.DispatcherServlet        : Completed initialization in 5 ms
2019-08-05 17:04:21.488  INFO 19548 --- [nioEventLoopGroup-3-1] c.kang.netty.handler.ServerFrameHandler  : 23:老王
2019-08-05 17:04:21.622  INFO 19548 --- [nioEventLoopGroup-3-1] c.kang.netty.handler.ServerFrameHandler  : 23:老王
2019-08-05 17:04:21.769  INFO 19548 --- [nioEventLoopGroup-3-1] c.kang.netty.handler.ServerFrameHandler  : 23:老王
2019-08-05 17:04:21.915  INFO 19548 --- [nioEventLoopGroup-3-1] c.kang.netty.handler.ServerFrameHandler  : 23:老王
2019-08-05 17:17:10.900  WARN 19548 --- [nioEventLoopGroup-2-1] io.netty.channel.AbstractChannel         : Can't invoke task later as EventLoop rejected it

java.util.concurrent.RejectedExecutionException: event executor terminated
	at io.netty.util.concurrent.SingleThreadEventExecutor.reject(SingleThreadEventExecutor.java:842) ~[netty-all-4.1.25.Final.jar:4.1.25.Final]
	at io.netty.util.concurrent.SingleThreadEventExecutor.offerTask(SingleThreadEventExecutor.java:328) ~[netty-all-4.1.25.Final.jar:4.1.25.Final]
	at io.netty.util.concurrent.SingleThreadEventExecutor.addTask(SingleThreadEventExecutor.java:321) ~[netty-all-4.1.25.Final.jar:4.1.25.Final]
	at io.netty.util.concurrent.SingleThreadEventExecutor.execute(SingleThreadEventExecutor.java:765) ~[netty-all-4.1.25.Final.jar:4.1.25.Final]
	at io.netty.channel.AbstractChannel$AbstractUnsafe.invokeLater(AbstractChannel.java:1012) [netty-all-4.1.25.Final.jar:4.1.25.Final]
	at io.netty.channel.AbstractChannel$AbstractUnsafe.deregister(AbstractChannel.java:813) [netty-all-4.1.25.Final.jar:4.1.25.Final]
	at io.netty.channel.AbstractChannel$AbstractUnsafe.fireChannelInactiveAndDeregister(AbstractChannel.java:773) [netty-all-4.1.25.Final.jar:4.1.25.Final]
	at io.netty.channel.AbstractChannel$AbstractUnsafe.close(AbstractChannel.java:756) [netty-all-4.1.25.Final.jar:4.1.25.Final]
	at io.netty.channel.AbstractChannel$AbstractUnsafe.close(AbstractChannel.java:611) [netty-all-4.1.25.Final.jar:4.1.25.Final]
	at io.netty.channel.nio.NioEventLoop.closeAll(NioEventLoop.java:695) [netty-all-4.1.25.Final.jar:4.1.25.Final]
	at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:474) [netty-all-4.1.25.Final.jar:4.1.25.Final]
	at io.netty.util.concurrent.SingleThreadEventExecutor$5.run(SingleThreadEventExecutor.java:884) [netty-all-4.1.25.Final.jar:4.1.25.Final]
	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30) [netty-all-4.1.25.Final.jar:4.1.25.Final]
	at java.lang.Thread.run(Thread.java:748) [na:1.8.0_181]

2019-08-05 17:17:10.900  WARN 19548 --- [nioEventLoopGroup-2-2] io.netty.channel.AbstractChannel         : Can't invoke task later as EventLoop rejected it

java.util.concurrent.RejectedExecutionException: event executor terminated
	at io.netty.util.concurrent.SingleThreadEventExecutor.reject(SingleThreadEventExecutor.java:842) ~[netty-all-4.1.25.Final.jar:4.1.25.Final]
	at io.netty.util.concurrent.SingleThreadEventExecutor.offerTask(SingleThreadEventExecutor.java:328) ~[netty-all-4.1.25.Final.jar:4.1.25.Final]
	at io.netty.util.concurrent.SingleThreadEventExecutor.addTask(SingleThreadEventExecutor.java:321) ~[netty-all-4.1.25.Final.jar:4.1.25.Final]
	at io.netty.util.concurrent.SingleThreadEventExecutor.execute(SingleThreadEventExecutor.java:765) ~[netty-all-4.1.25.Final.jar:4.1.25.Final]
	at io.netty.channel.AbstractChannel$AbstractUnsafe.invokeLater(AbstractChannel.java:1012) [netty-all-4.1.25.Final.jar:4.1.25.Final]
	at io.netty.channel.AbstractChannel$AbstractUnsafe.deregister(AbstractChannel.java:813) [netty-all-4.1.25.Final.jar:4.1.25.Final]
	at io.netty.channel.AbstractChannel$AbstractUnsafe.fireChannelInactiveAndDeregister(AbstractChannel.java:773) [netty-all-4.1.25.Final.jar:4.1.25.Final]
	at io.netty.channel.AbstractChannel$AbstractUnsafe.close(AbstractChannel.java:756) [netty-all-4.1.25.Final.jar:4.1.25.Final]
	at io.netty.channel.AbstractChannel$AbstractUnsafe.close(AbstractChannel.java:611) [netty-all-4.1.25.Final.jar:4.1.25.Final]
	at io.netty.channel.nio.NioEventLoop.closeAll(NioEventLoop.java:695) [netty-all-4.1.25.Final.jar:4.1.25.Final]
	at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:474) [netty-all-4.1.25.Final.jar:4.1.25.Final]
	at io.netty.util.concurrent.SingleThreadEventExecutor$5.run(SingleThreadEventExecutor.java:884) [netty-all-4.1.25.Final.jar:4.1.25.Final]
	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30) [netty-all-4.1.25.Final.jar:4.1.25.Final]
	at java.lang.Thread.run(Thread.java:748) [na:1.8.0_181]

2019-08-05 17:17:10.900  WARN 19548 --- [nioEventLoopGroup-3-1] io.netty.channel.AbstractChannel         : Can't invoke task later as EventLoop rejected it

java.util.concurrent.RejectedExecutionException: event executor terminated
	at io.netty.util.concurrent.SingleThreadEventExecutor.reject(SingleThreadEventExecutor.java:842) ~[netty-all-4.1.25.Final.jar:4.1.25.Final]
	at io.netty.util.concurrent.SingleThreadEventExecutor.offerTask(SingleThreadEventExecutor.java:328) ~[netty-all-4.1.25.Final.jar:4.1.25.Final]
	at io.netty.util.concurrent.SingleThreadEventExecutor.addTask(SingleThreadEventExecutor.java:321) ~[netty-all-4.1.25.Final.jar:4.1.25.Final]
	at io.netty.util.concurrent.SingleThreadEventExecutor.execute(SingleThreadEventExecutor.java:765) ~[netty-all-4.1.25.Final.jar:4.1.25.Final]
	at io.netty.channel.AbstractChannel$AbstractUnsafe.invokeLater(AbstractChannel.java:1012) [netty-all-4.1.25.Final.jar:4.1.25.Final]
	at io.netty.channel.AbstractChannel$AbstractUnsafe.deregister(AbstractChannel.java:813) [netty-all-4.1.25.Final.jar:4.1.25.Final]
	at io.netty.channel.AbstractChannel$AbstractUnsafe.fireChannelInactiveAndDeregister(AbstractChannel.java:773) [netty-all-4.1.25.Final.jar:4.1.25.Final]
	at io.netty.channel.AbstractChannel$AbstractUnsafe.close(AbstractChannel.java:756) [netty-all-4.1.25.Final.jar:4.1.25.Final]
	at io.netty.channel.AbstractChannel$AbstractUnsafe.close(AbstractChannel.java:611) [netty-all-4.1.25.Final.jar:4.1.25.Final]
	at io.netty.channel.nio.NioEventLoop.closeAll(NioEventLoop.java:695) [netty-all-4.1.25.Final.jar:4.1.25.Final]
	at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:474) [netty-all-4.1.25.Final.jar:4.1.25.Final]
	at io.netty.util.concurrent.SingleThreadEventExecutor$5.run(SingleThreadEventExecutor.java:884) [netty-all-4.1.25.Final.jar:4.1.25.Final]
	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30) [netty-all-4.1.25.Final.jar:4.1.25.Final]
	at java.lang.Thread.run(Thread.java:748) [na:1.8.0_181]

2019-08-05 17:17:54.742  INFO 17008 --- [main] cn.kang.my_netty.BootTest                : Starting BootTest on PS20190426HEJH with PID 17008 (started by Admin in D:\workSpace\IDEACode\my_netty)
2019-08-05 17:17:54.743  INFO 17008 --- [main] cn.kang.my_netty.BootTest                : No active profile set, falling back to default profiles: default
2019-08-05 17:17:55.502  INFO 17008 --- [main] cn.kang.config.RedisConfig               : JedisPool注入成功！！
2019-08-05 17:17:55.502  INFO 17008 --- [main] cn.kang.config.RedisConfig               : redis地址：114.116.175.209:6379
2019-08-05 17:17:56.471  INFO 17008 --- [main] .s.a.AnnotationAsyncExecutionInterceptor : No task executor bean found for async processing: no bean of type TaskExecutor and no bean named 'taskExecutor' either
2019-08-05 17:17:56.481  INFO 17008 --- [main] cn.kang.my_netty.BootTest                : Started BootTest in 1.988 seconds (JVM running for 2.635)
2019-08-05 17:28:20.144  INFO 14252 --- [main] cn.kang.my_netty.BootTest                : Starting BootTest on PS20190426HEJH with PID 14252 (started by Admin in D:\workSpace\IDEACode\my_netty)
2019-08-05 17:28:20.144  INFO 14252 --- [main] cn.kang.my_netty.BootTest                : No active profile set, falling back to default profiles: default
2019-08-05 17:28:20.833  INFO 14252 --- [main] cn.kang.config.RedisConfig               : JedisPool注入成功！！
2019-08-05 17:28:20.833  INFO 14252 --- [main] cn.kang.config.RedisConfig               : redis地址：114.116.175.209:6379
2019-08-05 17:28:21.833  INFO 14252 --- [main] .s.a.AnnotationAsyncExecutionInterceptor : No task executor bean found for async processing: no bean of type TaskExecutor and no bean named 'taskExecutor' either
2019-08-05 17:28:21.839  INFO 14252 --- [main] cn.kang.my_netty.BootTest                : Started BootTest in 1.913 seconds (JVM running for 2.482)
